dashboard/src/reducers/authReducer.js

```diff
@@ -1,49 +1,20 @@
-import { KEEP_USER_LOGGED_IN,
-    USER_NOTION_ALERTS,
-    SET_SIGNUP_BUTTON,
-    SET_LOGIN_BUTTON,
-    SET_LOGIN_DETAILS, } from "../actions/types";
+import { USER_NOTION_ALERTS } from "../actions/types";
 
 const initialState = {
-    keepLoggedIn : false,
-    alerts: [],
-    isLoginBtnDisabled: true,
-    isSignupBtnDisabled: true,
-    passwordLength: 8,
-    loginDetails: {}
-}
+  alerts: [],
+};
 
 const AuthReducer = (state = initialState, action = {}) => {
-    const { type, payload } = action;
-    switch (type) {
-        case KEEP_USER_LOGGED_IN:
-            return {
-                ...state,
-                keepLoggedIn: payload
-            }
-        case USER_NOTION_ALERTS:
-            return {
-                ...state,
-                alerts: [...payload]
-            }
-        case SET_LOGIN_BUTTON:
-            return {
-                ...state,
-                isLoginBtnDisabled: payload
-            }
-        case SET_SIGNUP_BUTTON:
-            return {
-                ...state,
-                isSignupBtnDisabled: payload
-            }
-        case SET_LOGIN_DETAILS:
-            return {
-                ...state,
-                loginDetails: {...payload}
-            }
-        default:
-            return state;
-    }
-}
+  const { type, payload } = action;
+  switch (type) {
+    case USER_NOTION_ALERTS:
+      return {
+        ...state,
+        alerts: [...payload],
+      };
```

Comment:
`USER_NOTION_ALERTS` is used in only one place, with a payload of `[]`, which I think clears the `authReducer.alerts` list.  However, I don't see any code which adds to or displays that list, so I'm not sure what the point of clearing it is....

So, I think you can remove this whole file, and the definition of `USER_NOTION_ALERTS`, if you remove the `dispatch()` call from `movePage()` at line 50 in `dashboard/src/actions/authActions.js` (which, as I said, doesn't accomplish anything AFAICT).

(If you want to continue pulling on this thread, I think you could also get rid of `movePage()` by in-lining it's one remaining code line into its callers....)